gulpfile.js

```diff
@@ -296,11 +227,6 @@ gulp.task('watch-manifest', function () {
   }));
 });
 
-gulp.task('start-live-reload-server', function () {
-  var LiveReloadServer = require('./scripts/gulp/live-reload-server');
-  liveReloadServer = new LiveReloadServer(3000, 'http://localhost:5000');
-});
-
 gulp.task('build-app',
```

Comment:
Either in this PR or separately it would make sense to replace mentions of `app` with `site`, since we're no longer building the client here.